src/Homework1.java

```diff
@@ -1,101 +1,272 @@
+                continue;
+            else
+                System.out.print(" ");
+        }
+
+    }
+    public static void Problem7()
+    {
+        int[] v = {13, 20, 14, 5, 2, 8};
+        int minV = v[0];
+        int maxV = minV;
+        int sum = minV;
+        double avg;
+        for(int i = 1; i < v.length; ++i) {
+            sum += v[i];
+            if(v[i] < minV) {
+                minV = v[i];
+            }
+            if(v[i] > maxV) {
+                maxV = v[i];
+            }
+
+        }
+        avg = (double) sum / v.length;
+        System.out.println("MaxV: " + maxV);
+        System.out.println("MinV: " + minV);
+        System.out.println("Avg: " + avg);
+
+    }
+    public static void Problem8()
+    {
+        boolean iFoundIt = false;
+        int searchingValue = 5;
+        int[] v = { 13, 20, 14, 2, 8, 5, 5 };
+        for(int i = 0; i < v.length; ++i) {
+            if(v[i] == searchingValue) {
+                iFoundIt = true;
+                System.out.println("I found " + searchingValue + " at position " + i);
+            }
+        }
+        if(iFoundIt == false) {
+            System.out.println("The value " + searchingValue + " was not found");
+        }
 
-        j = 0;
-        do {
-            sum = sum + ints[j];
-            j = j + 1;
-        } while (j < ints.length);
-        System.out.println("Veniturile dupa do-while: " + sum);
+    }
+    public static void Problem9()
+    {
+        int[] v = { 13, 20, 14, 5, 2, 8 };
+        int evenCounter = 0;
+        int oddCounter = 0;
+        for(int i = 0; i < v.length; ++i) {
+            if(v[i] % 2 == 0){
+                evenCounter ++;
+            }
+            else{
+                oddCounter++;
+            }
+        }
+        System.out.println("Odd elements count: " + oddCounter);
+        System.out.println("Even elements count: " + evenCounter);
+    }
+    public static void Problem10()
+    {
+        String s1 = "Hello java world";
+        String s2 = "zorld";
+        boolean isS2inS1;
+        isS2inS1 = s1.contains(s2);
+        System.out.println(isS2inS1);
+    }
+    public static void Problem11()
+    {
+        String s = "Hello java world";
+        char c = 'a';
+        int noOfCinS = 0;
+        for(int i = 0; i < s.length(); ++i) {
+            if(s.charAt(i) == c){
+                noOfCinS++;
+            }
+        }
+        System.out.println(noOfCinS);
+    }
+    public static void Problem12()
+    {
+        String s1 = "Hello java world";
+        String s2 = "java";
+        boolean isSufix;
+        isSufix = s1.endsWith(s2);
+        System.out.println(isSufix);
+    }
+    public static void Problem13()
+    {
+        String s = "2abcd12";
+        int aux;
+        int ch;
+        for(int i = 0; i < s.length(); ++i){
+            ch = s.charAt(i);
+            if(ch < 'a' || ch > 'z') {
+                continue;
+            }
+            aux = ch - 'a';
```

Comment:
Frumos dar, ar fi fost interesant sa tratezi si literele mari. in cazul asta pentru un `A` rezultatul nu o sa fie la fel

Reply:
Updated.